Source/Core/Core/HW/EXI_DeviceGecko.cpp

```diff
@@ -48,14 +48,14 @@ void GeckoSockServer::GeckoConnectionWaiter()
 
 	sf::TcpListener server;
 	server_port = 0xd6ec; // "dolphin gecko"
-	for (int bind_tries = 0; bind_tries <= 10 && !server_running; bind_tries++)
+	for (int bind_tries = 0; bind_tries <= 10 && !server_running.load(); bind_tries++)
 	{
-		server_running = server.listen(server_port) == sf::Socket::Done;
-		if (!server_running)
+		server_running.store(server.listen(server_port) == sf::Socket::Done);
+		if (!server_running.load())
```

Comment:
Would this make more sense/make a difference if we used a local ```bool``` to catch the result of ```listen() == Done```, ```store()``` that local, and then use the local to kick of ```port++``` instead of doing a ```load()```  again? The only other place calling ```store()``` is the dtor, for which i believe it makes no difference here since the loop terminates on a ```load()``` too anyways.